server/routes/referrals/complexityLevelPresenter.test.ts

```diff
@@ -80,7 +58,7 @@ describe('ComplexityLevelPresenter', () => {
     describe('when the referral already has a selected complexity level and there is user input data', () => {
       it('sets checked to true for the complexity level that the user chose', () => {
         draftReferral.complexityLevelId = 'd0db50b0-4a50-4fc7-a006-9c97530e38b2'
-        const presenter = new ComplexityLevelPresenter(draftReferral, socialInclusionComplexityLevels, null, {
+        const presenter = new ComplexityLevelPresenter(draftReferral, serviceCategory, null, {
           'complexity-level': '110f2405-d944-4c15-836c-0c6684e2aa78',
```

Comment:
To decrease dependence on the exact values in the factory

```suggestion
          'complexity-level': serviceCategory.complexityLevels[1],
```